biz.aQute.bndlib/src/aQute/bnd/service/Tags.java

```diff
@@ -0,0 +1,65 @@
+package aQute.bnd.service;
+
+import static java.util.stream.Collectors.toCollection;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.LinkedHashSet;
+import java.util.Set;
+
+/**
+ * A set of tags. A tag is a string-token which can be attached to an entity for
+ * categorization and filtering. Typically these entities then implement the
+ * {@link Tagged} interface.
+ */
+public final class Tags extends LinkedHashSet<String> {
```

Comment:
Not a good idea to extend this class because it is read-write. I'd use an internal set and implement Set<String>. Also make sure the strings are sorted to keep the build repeatable.

Reply:
Ok I will change it and implement set using a SortedSet internally.

Reply:
Added https://github.com/bndtools/bnd/pull/6116/commits/7857d51a22937b0a338938715dc826721b437154